### PR TITLE
BFD-1557: Introduce migrator runbook

### PR DESCRIPTION
**Pull request by mjburling**
**JIRA Ticket:**
[BFD-1557](https://jira.cms.gov/browse/BFD-1557)

**User Story or Bug Summary:**
Introduce migrator runbook

---

### What Does This PR Do?
Here, we're introducing a runbook that operators might reference when encountering problems during migrator deployments. In doing so, we can transition [RFC#0011](https://github.com/CMSgov/beneficiary-fhir-data/blob/master/rfcs/0011-separate-flyway-from-pipeline.md) to an **Implemented** status.

### What Should Reviewers Watch For?

If you're reviewing this PR, please check for these things in particular:

* Verify all PR security questions and checklists have been completed and addressed.

### What Security Implications Does This PR Have?

Submitters should complete the following questionnaire:

* If the answer to any of the questions below is **Yes**, then **you must** supply a link to the associated Security Impact Assessment (SIA), security checklist, or other similar document in Confluence here: **N/A**

    * Does this PR add any new software dependencies? 
      * [ ] Yes
      * [x] No
    * Does this PR modify or invalidate any of our security controls?
      * [ ] Yes
      * [x] No
    * Does this PR store or transmit data that was not stored or transmitted before?
      * [ ] Yes
      * [x] No

* If the answer to any of the questions below is **Yes**, then please add @StewGoin as a reviewer, and note that this PR **should not be merged** unless/until he also approves it.
    * Do you think this PR requires additional review of its security implications for other reasons?
      * [ ] Yes
      * [x] No

### What Needs to Be Merged and Deployed Before this PR?

This PR cannot be either merged or deployed until the following prerequisite changes have been fully deployed:

* N/A

### Submitter Checklist

I have gone through and verified that...:

* [x] I have named this PR and branch so they are [automatically linked](https://confluence.atlassian.com/adminjiracloud/integrating-with-development-tools-776636216.html) to the (most) relevant Jira issue. Ie: `BFD-123: Adds foo`
* [x] This PR is reasonably limited in scope, to help ensure that:
    1. It doesn't unnecessarily tie a bunch of disparate features, fixes, refactorings, etc. together.
    2. There isn't too much of a burden on reviewers.
    3. Any problems it causes have a small "blast radius".
    4. It'll be easier to rollback if that becomes necessary.
* [x] This PR includes any required documentation changes, including `README` updates and changelog / release notes entries.
* [x] The data dictionary has been updated with any field mapping changes, if any were made.
* [x] All new and modified code is appropriately commented, such that the what and why of its design would be reasonably clear to engineers, preferably ones unfamiliar with the project.
* [x] All tech debt and/or shortcomings introduced by this PR are detailed in `TODO` and/or `FIXME` comments, which include a JIRA ticket ID for any items that require urgent attention.
* [x] Reviews are requested from both:
    * At least two other engineers on this project, at least one of whom is a senior engineer or owns the relevant component(s) here.
    * Any relevant engineers on other projects (e.g. DC GEO, BB2, etc.).
* [x] Any deviations from the other policies in the [DASG Engineering Standards](https://github.com/CMSgov/cms-oeda-dasg/blob/master/policies/engineering_standards.md) are specifically called out in this PR, above.
    * Please review the standards every few months to ensure you're familiar with them.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -62,6 +62,7 @@ def gitCommitId
 def gitRepoUrl
 def awsRegion = 'us-east-1'
 def verboseMaven = params.verbose_mvn_logging
+def migratorRunbookUrl = "https://github.com/CMSgov/beneficiary-fhir-data/blob/master/runbooks/how-to-recover-from-migrator-failures.md"
```
Comment (mjburling) [inline Jenkinsfile:65]: This is a low risk change, but I'd **still** like to fully test this prior to merging.

Comment (mjburling) [reply Jenkinsfile:65]: OK!

1. Out of band, I created the migrator instance in `test`, and left a populated SQS queue in tact (attempting to trigger the Undeployable State error).
2. I attempted to deploy to `test` via Jenkins on PR-1475 and I was met with the expected error and link to a runbook (although it's hardcoded to master, which doesn't yet exist...)
3. Using the runbook corrective steps, I've re-deployed to `test` via Jenkins, and the deployment is functioning as expected.

**Review by malessi — APPROVED** [review]
:100:

File: rfcs/0011-separate-flyway-from-pipeline.md [hunk 2]
```diff
@@ -15,9 +15,9 @@ and deploying migrations more robust and efficient.
 ## Status
 [Status]: #status
 
-* Status: Proposed
+* Status: Implemented
 * Implementation JIRA Ticket(s):
-  * [BFD-XXXX](https://jira.cms.gov/browse/BFD-XXXX)
+  * [BFD-1560](https://jira.cms.gov/browse/BFD-1560)
```
Comment (christrotteradhoc) [inline rfcs/0011-separate-flyway-from-pipeline.md:20]: 💯

**Review by christrotteradhoc — APPROVED** [review]

**Pull request merged**